# Peak metrics. 

Note: The peak calls analyzed here are output from a uniform processing pipeline (https://github.com/epigen-UCSD/atac_seq_pipeline), and have not been optimized for your samples. In some cases, parameter tuning is required to achieve optimal peak call results. We recommend that you closely examine peak calls on a genome browser together with your signal tracks (*.bigwig) before using in downstream analysis. It may be necessary for you to test different parameters and re-call peaks prior to downstream analysis. The Center can give advice on best practices, but, unfortunately, does not have the bandwidth to perform downstream   analysis   in  most  cases.   For  additional  bioinformatics   support,   we  recommend contacting the UCSD Center for Computational Biology & Bioinformatics (http://compbio.ucsd.edu/) which can provide ATAC-seq analysis services. 

1. [Peak counts](05a_peakcounts.md). 
2. [Fraction of Reads in Peaks (FRiP)](05b_FRiP.md). 
3. [PCA](05c_pca.md).
4. [Correlation matrix](05d_cor.md).
